src/components/InteractiveMap/InteractiveMap.vue

```diff
@@ -105,105 +52,125 @@
 <script lang="ts">
 import Vue from "vue";
 import axios from "axios";
-/// <reference path="@/types/escher.d.ts" />
-import * as escher from "@dd-decaf/escher";
 import * as settings from "@/settings";
+import Escher from "@/components/InteractiveMap/Escher.vue";
+import Card from "@/components/InteractiveMap/Card.vue";
 
 export default Vue.extend({
   name: "InteractiveMap",
+  components: {
+    Escher,
+    Card
+  },
   data: () => ({
     escherBuilder: null,
-    isInitializingEscher: true,
     currentMapId: null,
-    isLoadingMap: false,
-    cards: [
-      {
-        // TODO: Replace test card
-        name: "Design",
-        method: "pFBA",
-        model: { name: "iJO1366" },
-        organism: { name: "E. coli" },
-        growthRate: 0.874
-      }
-    ]
+    fluxDistribution: null,
+    mapData: null,
+    cards: []
   }),
   methods: {
+    escherLoaded() {
```

Comment:
Confusion on my part: Why is this a method and not a computed property?

Reply:
This is an event listener that runs when escher is initialized. It calls the `addDefaultCard` method - but makes sure to do it only when models are available in the store (hence passing it through `$store.getters["models/onData"]`).

It does not return any data, so it's not a computed property.